spec/services/webhook_service_spec.rb

```diff
@@ -58,6 +58,25 @@
       expect(payload['username']).to eq 'Loomio Bot'
       expect(payload['text']).to match /#{user.name}.*updated the outcome for.*#{motion.name}/
     end
+
+    it 'serializes a new vote' do 
+      event = VoteService.create actor: user, vote: vote
+      def vote_position
```

Comment:
Hi James, we weren't sure how to get access to the appropriate vote_position text in the test, so we put the method in there to pass the test for the time being. It would be awesome to find a better way around this! We're working on it right now.